src/include/crowbar/complex.rb

```diff
@@ -741,18 +741,22 @@ def InitRPRepos(id)
               elsif url.include? "ks/dist/child"
                 @repos_location = "sm"
                 @remote_server_url = url.gsub(/(^.*)\/ks\/dist\/child\/.*/,"\\1")
+              elsif !url.empty?
+                @repos_location = "custom"
               end
-              break unless @repos_location == "custom"
+              break unless @repos_location == "local"
             end
-            break unless @repos_location == "custom"
+            break unless @repos_location == "local"
```

Comment:
Hm, not sure if it would be simplification. I would gain one more nesting, and had to throw in 3 cases (as compared with 2 cases of break)